bengine:
 
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
 
  # Position Controllers ---------------------------------------
  LF_position_controller:
    type: position_controllers/JointPositionController
    joint: LF_JOINT
    #pid: {p: 1.0, i: 0.01, d: 0.0}

  LB_position_controller:
    type: position_controllers/JointPositionController
    joint: LB_JOINT
    #pid: {p: 1.0, i: 0.01, d: 0.0}

  RF_position_controller:
    type: position_controllers/JointPositionController
    joint: RF_JOINT
    #pid: {p: 1.0, i: 0.01, d: 0.0}

  RB_position_controller:
    type: position_controllers/JointPositionController
    joint: RB_JOINT
    #pid: {p: 1.0, i: 0.01, d: 0.0}
 
 
 
bengine/gazebo_ros_control/pid_gains:
  LF_JOINT: {p: 1.0, i: 0.0, d: 0.0}
  LB_JOINT: {p: 1.0, i: 0.0, d: 0.0}
  RF_JOINT: {p: 1.0, i: 0.0, d: 0.0}
  RB_JOINT: {p: 1.0, i: 0.0, d: 0.0}
